com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineGlobalSettings.cs

```diff
@@ -502,18 +333,32 @@ void EnsureResources<T>(bool forceReload, ref T resources, string resourcePath,
                 if (forceReload)
                     ResourceReloader.ReloadAllNullIn(resources, HDUtils.GetHDRenderPipelinePath());
             }
-            else // Asset database may not be ready
+            else
             {
+                // Asset database may not be ready
                 var objs = InternalEditorUtility.LoadSerializedFileAndForget(resourcePath);
                 resources = (objs != null && objs.Length > 0) ? objs[0] as T : null;
                 if (forceReload)
```

Comment:
you cannot go through that code path since you are already in !forceReload

Reply:
Well no we are not o_o